crates/rattler/src/install/python.rs

```diff
@@ -68,6 +68,23 @@ impl PythonInfo {
         &self.path
     }
 
+    /// Constructs a shebang that will run the rest of the script as Python.
+    pub fn shebang(&self, target_prefix: &str) -> String {
+        let target_path = Path::new(target_prefix).join(self.path());
+        let target_path = target_path.as_os_str().to_string_lossy();
+
+        // Shebangs cannot be larger than 127 characters and executables with spaces are
+        // problematic.
+        if target_path.len() > 127 - 2 {
```

Comment:
I think we should also check if the `target_path` contains a space here (as the comment indicates :))